splinter/driver/webdriver/remote.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, url=DEFAULT_URL, browser="firefox", wait_time=2, **ability_ar
         abilities = getattr(DesiredCapabilities, browsername, {})
         abilities.update(ability_args)
 
-        self.driver = Remote(url, abilities)
+        self.driver = Remote(url, abilities, keep_alive=True)
```

Comment:
I believe that a better way is to receive a parameter that defines if the keep_alive is enabled (or not)

Reply:
Sure. I've opened #734 with more expansive changes, but will match all of Remote WebDriver's arguments.